pkg/ocm/storage/outcoming/ocm.go

```diff
@@ -266,22 +267,27 @@ func (d *driver) unwrappedOpFromShareCreator(ctx context.Context, share *ocmv1be
 }
 
 func (d *driver) GetMD(ctx context.Context, ref *provider.Reference, _ []string) (*provider.ResourceInfo, error) {
+	log := appctx.GetLogger(ctx)
 	share, rel, err := d.shareAndRelativePathFromRef(ctx, ref)
 	if err != nil {
+		log.Error().Err(err).Msg("Error from shareAndRelativePathFromRef")
```

Comment:
instead of logging errors here as we might have duplicates from the caller, we should pass to the called a wrapped error.